olewoo_installer/Setup.wxs

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Wix xmlns="http://schemas.microsoft.com/wix/2006/wi" xmlns:netfx="http://schemas.microsoft.com/wix/NetFxExtension">
+    <?define UPGRADE_CODE="B6E28170-3F1B-44A6-AFC8-3F4B82A20F37"?>
+  <?define OUTPUT_CAB="olewoo"?>
+  <?define SOURCE="..\\$(var.Configuration)"?>
+  <?define PRODUCT_NAME = "OLEWOO"?>
+  <?define PRODUCT_DESC = "Type Library Viewer"?>
+  <?define MANUFACTURER="Liebnitz"?>
```

Comment:
L_ei_bnitz after Gottfried Wilhelm, ;)   But might be easier if you just put github url as manufacturer?
I know it's not quite canon, but works.

Is there any reason it's all in capitals?  I don't know about you but I find caps really annoying in application names/paths.  (Not a huge deal, but while I'm here!)

Reply:
Personally, I don't like pascal cased acronyms .e.g Ole, Sql, Xml. I don't even know what "woo" is, other than something that men did to women in days of yore.

But it's your project. I'll change it to OleWoo like you have in the title bar of the UI.

I'll just make it the way you prefer. Would you like a license acceptance screen (I'd use the first pre element from license.txt? This could also lead to choosing background and banner images.

Reply:
'Woo'? Honestly, it was just the first vaguely homophonic thing that sprang to mind when slavishly reimplementing *cough* oleview.   There's also a vague connotation of success.   I was not cut out for marketing.

Casing - TBH I naturally favour all lower, as I spend 90% of my time in unixland, but I'm happy with CaMel or Initialcaps, as those seem to be the Windows standards. (from a brief browse of my progra~1).

Licence - Genuinely not bothered.  ;)

Reply:
I found out how to do the manufacturerUrl  - there's a property you have to set. This makes no difference to Settings/Apps but it does to Programs and Features:

![image](https://user-images.githubusercontent.com/29954900/82834062-e2d47680-9eb7-11ea-982a-e9fece399601.png)